06-building-a-program-start-to-finish/solving-the-problem/Output/invertedIndex.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Inverted index class
+ */
+class InvertedIndex {
+  /**
+   * Inverted index constructor
+   */
+  constructor() {
+    // Object to hold the index
+    this.index = {};
+  }
+
+  /**
```

Comment:
There's no description for the `tokenize` method. I don't know what it's supposed to do.

Please fix for all your methods.